File: AccuPay/Domain/Employees/TardinessRecord.vb
Option Strict On
Imports System.ComponentModel.DataAnnotations.Schema

Namespace Global.AccuPay.Entity

    <Table("tardinessrecord")>
    Public Class TardinessRecord

        Public Property Year As Integer
        Public Property EmployeeId As Integer
        Public Property FirstOffenseDate As Date

    End Class

End Namespace

File: Accupay.Data/Entities/TardinessRecord.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace AccuPay.Data.Entities
{
    [Table("tardinessrecord")]
    public class TardinessRecord
    {
        public int Year { get; set; }
        public int EmployeeId { get; set; }
        public DateTime FirstOffenseDate { get; set; }
    }
}